Feat(eos_designs)!: Update SNMP EngineID algorithms (#5576)
aristanetworks/ansible-avd: docs/porting-guides/6.x.x.md

+ underlay_ospf_graceful_restart: false
```

### The default value for `snmp_settings.compute_local_engineid_source` is now `rfc3411`

> In AVD 5.x, the default value of `snmp_settings.compute_local_engineid_source` was `hostname_and_ip`. This had several issues,

Reviewer: ```suggestion
In AVD 5.x, the default value of `snmp_settings.compute_local_engineid_source` was `hostname_and_ip`. This method had several issues;
```

> first this is not managing well inband management scenarii and secondly this is not compliant with RFC 3411 format.

Reviewer: ```suggestion
firstly the method is not using the inband management IP, even if no management interface IP is set, and secondly the resulting Engine ID is not compliant with the RFC 3411 format.
```

> Starting with AVD 6.0.0, the default value has been changed to `rfc3411`. This now generates a RFC 3411 compliant engine ID and handles the inband management IP cases gracefully.

Reviewer: ```suggestion
Starting with AVD 6.0.0, the default value has been changed to `rfc3411`. This now generates a RFC 3411 compliant engine ID and handles the inband management IP cases properly.
```

+ underlay_ospf_graceful_restart: false
```

### The default value for `snmp_settings.compute_local_engineid_source` is now `rfc3411`

In AVD 5.x, the default value of `snmp_settings.compute_local_engineid_source` was `hostname_and_ip`. This method had several issues;
firstly the method is not using the inband management IP, even if no management interface IP is set, and secondly the resulting Engine ID is not compliant with the RFC 3411 format.

> Starting with AVD 6.0.0, the default value has been changed to `rfc3411`. This now generates a RFC 3411 compliant engine ID and handles the inband management IP cases properly.

Reviewer 2: ```suggestion
Starting with AVD 6.0.0, the default value has been changed to `rfc3411_type5`. This now generates a RFC 3411 Type 5 compliant engine ID and handles the inband management IP cases properly.
```
